pkg/models/trb_request.go

```diff
@@ -44,8 +44,8 @@ type TRBRequestState string
 
 // These are the options for TRBRequestStatus
 const (
-	TRBRequestStateOpen   TRBRequestState = "OPEN"
-	TRBRequestStateClosed TRBRequestState = "CLOSED"
+	TRBRequestStateOPEN   TRBRequestState = "OPEN"
+	TRBRequestStateCLOSED TRBRequestState = "CLOSED"
```

Comment:
Changes in the PR look good -- though I might prefer the name how it was before (usually capitalization like that is left for [acronyms/initialisms](https://go.dev/wiki/CodeReviewComments#initialisms) in variable names, like how `TRB` is as it is in the name, rather than `Trb`)

I won't request changes on it, since it doesn't really block the PR/functionality, but might prefer the older name!

Reply:
all good! i did it to match system intake, i will lowercase that one 😄

Reply:
Ah, interesting catch! Yeah, I'd prefer we lowercase that one!

https://github.com/CMSgov/easi-app/pull/2604/files#diff-49b06d592040dbe1da1e493b943e72d120c71304bb97f92b9974025ea5d6d71aL38

This is another example of why I'd prefer that casing -- otherwise it'd have to be `TRBTFORMALREVIEW`, or something along that line

Reply:
Makes sense! We could standardize those to `TRBRequestTypeNeedHelp` to match the layout of the other consts nearby

Reply:
https://github.com/CMSgov/easi-app/pull/2604/commits/c1fd85ca123e090e11071196fc598e290eb5ea9e

Reply:
Thanks!